types/node/index.d.ts

```diff
@@ -2833,14 +2833,17 @@ declare module "fs" {
      */
     export function readFileSync(filename: string, options?: { flag?: string; }): Buffer;
     export function writeFile(filename: string, data: any, callback?: (err: NodeJS.ErrnoException) => void): void;
+    export function writeFile(filename: string, data: any, encoding: string, callback?: (err: NodeJS.ErrnoException) => void): void;
```

Comment:
OK. I'll change callback in **new** definitions only in this PR.
As @bjouhier says, the *options is string* case should be fixed soon.